frontend/public/components/secret.jsx

```diff
@@ -34,17 +36,20 @@ const actionButtons = [addSecretToWorkload];
 const menuActions = [
   Kebab.factory.ModifyLabels,
   Kebab.factory.ModifyAnnotations,
-  (kind, obj) => ({
-    label: `Edit ${kind.label}`,
-    href: `${resourceObjPath(obj, kind.kind)}/edit`,
-    accessReview: {
-      group: kind.apiGroup,
-      resource: kind.plural,
-      name: obj.metadata.name,
-      namespace: obj.metadata.namespace,
-      verb: 'update',
-    },
-  }),
+  (kind, obj) => {
+    const { t } = useTranslation();
```

Comment:
You shouldn't be calling `useTranslation` here. It's only to be used inside a component. Return a `labelKey` instead.